### DISCUSSION TITLE
unable to verify your academic affiliation

### DISCUSSION THREAD

**Discussion by HonorPosession** [community/community#113793] [GitHub Education]
### Hi everyone,

Hi. i successfully submitted my School id (serbian: "indeks") after 15 unsuccessful attempts, probably because the ID is in cyrillic. But after finally doing so i was excited until i received this email yesterday:

"...Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Mar 21, 2024..."

I would like to know where the issue is, i am located like 40ish kilometers from my school, i am first year of Information Technology and I know i go there. It is very frustrating that after providing the MAIN document that **ABSOLUTELY PROVES** that you go to a school, with an official stamp and the principals signature, i am somehow not proven to go to that school. the school is recognized by github clearly since i am able to select it. what can be the reason? the final image i uploaded even had a paper next to it with my name written twice in Latin so the automated system can barely recognize it. I would love if someone would manually look at my document that i can RECORD LIVE if needed just to get accepted.

**Comment by samus-aran** [comment]
Hi @HonorPosession 

Usually with that message if there is no other rejection reason then try and submit a live photo of your evidence rather than an upload. You may include multiple documents in your image including the translation. Perhaps print out the translation if you have small handwriting (I know I do..).
